[2024-09-10T04:49:26.821+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-10T04:49:26.861+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_product_bash.say_welcome scheduled__2024-09-09T00:00:00+00:00 [queued]>
[2024-09-10T04:49:26.882+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_product_bash.say_welcome scheduled__2024-09-09T00:00:00+00:00 [queued]>
[2024-09-10T04:49:26.883+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-09-10T04:49:26.921+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): say_welcome> on 2024-09-09 00:00:00+00:00
[2024-09-10T04:49:26.932+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=512) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-10T04:49:26.935+0000] {standard_task_runner.py:72} INFO - Started process 513 to run task
[2024-09-10T04:49:26.934+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'customer_product_bash', 'say_welcome', 'scheduled__2024-09-09T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/retailData_dags.py', '--cfg-path', '/tmp/tmp9k_q0az3']
[2024-09-10T04:49:26.936+0000] {standard_task_runner.py:105} INFO - Job 10: Subtask say_welcome
[2024-09-10T04:49:27.020+0000] {task_command.py:467} INFO - Running <TaskInstance: customer_product_bash.say_welcome scheduled__2024-09-09T00:00:00+00:00 [running]> on host 793733a87fcc
[2024-09-10T04:49:27.189+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='muhii.smta@gmail.com' AIRFLOW_CTX_DAG_OWNER='Muhii' AIRFLOW_CTX_DAG_ID='customer_product_bash' AIRFLOW_CTX_TASK_ID='say_welcome' AIRFLOW_CTX_EXECUTION_DATE='2024-09-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-09T00:00:00+00:00'
[2024-09-10T04:49:27.191+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-10T04:49:27.272+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-10T04:49:27.274+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hallo !"']
[2024-09-10T04:49:27.293+0000] {subprocess.py:86} INFO - Output:
[2024-09-10T04:49:27.296+0000] {subprocess.py:93} INFO - Hallo !
[2024-09-10T04:49:27.297+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-10T04:49:27.343+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-10T04:49:27.345+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=customer_product_bash, task_id=say_welcome, run_id=scheduled__2024-09-09T00:00:00+00:00, execution_date=20240909T000000, start_date=20240910T044926, end_date=20240910T044927
[2024-09-10T04:49:27.399+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-10T04:49:27.457+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-10T04:49:27.461+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
